pkg/actors/reminders/reminders.go

```diff
@@ -159,9 +160,19 @@ func (r *reminders) CreateReminder(ctx context.Context, reminder *internal.Remin
 
 	stop := make(chan struct{})
 
-	err = r.storeReminder(ctx, store, reminder, stop)
+	config := retry.DefaultConfig()
+	config.Multiplier = 1.0
+	b := config.NewBackOffWithContext(ctx)
+
+	err = retry.NotifyRecover(func() error {
+		return r.storeReminder(ctx, store, reminder, stop)
```

Comment:
I wonder if we should retry only if the error is related to etags? We have a way to see if the error is an etag error (which is implemented in components-contrib and has conformance tests): https://vscode.dev/github/ItalyPaleAle/dapr/blob/master/pkg/grpc/api.go#L703-L707

Reply:
makes sense - let me try that

Reply:
done